templates/support.html

```diff
@@ -5,211 +5,92 @@
-                    everyone. The team ensures that the changes work in practice and
-                    that they are in line with the Matrix Manifesto.
+                    One of the most efficient ways to help improve Matrix and make
+                    it reach the masses is to make a donation. We provide a neutral
+                    groud for the ecosystem to gather and discuss the specification,
+                    we provide a server to let less tech-savvy people join the
+                    federation, and we spread the word at events and conferences.
                 </p>
                 <p>
-                    The Specification is the building plan for everyone making Matrix
-                    products, and the Spec Core Team are its architects.
+                    If you believe in what we do and want to support us in our
+                    mission, a donation, even small, can go a long way.
                 </p>
             </div>
-            <img src="/assets/spec-logo.svg" alt="The Matrix logo with rulers apparent" />
+            <a class="call-to-action" href="https://donorbox.org/keep-matrix-exciting">
+                Donate with Donorbox
+            </a>
         </div>
     </div>
-    <div id="tangible-work">
+
+    <div id="membership">
         <div class="content">
-            <img src="/assets/frontpage/federation-visual-mobile.svg"
-                alt="An abstract representation of the federation" />
+            <h2>Join as a member</h2>
             <div>
-                <h2>Tangible work</h2>
                 <p>
-                    We're pragmatists who don't only work in theory. We want
-                    Matrix to succeed in real life, so we make it happen in real
-                    life. We provide two homeserver implementations (Synapse and
-                    Dendrite), SDKs for clients, bridges and integrations in
-                    various languages, provide the libraries implementing the
-                    encryption Matrix relies on and get them audited.
+                    If you have a stake in Matrix, become a member of the Foundation
+                    and run for the Governing Board to have a say in the direction
+                    of the project.
                 </p>
                 <p>
-                    The Matrix.org Foundation provides software people can use
-                    as is, and software people can use to make their own
-                    products based on Matrix.
+                    An organizational membership offers additional perks, such as a
+                    logo on our website and discounts when organizing our annual
+                    conference.
                 </p>
             </div>
+            <a href="/membership" class="call-to-action">Become a member</a>
         </div>
     </div>
-    <div id="all-your-comms">
+
+    <div id="working-group">
         <div class="content">
+            <h2>Join a working group</h2>
```

Comment:
maybe worth mentioning wrt volunteering, I think the SCT has also repeatedly said you can support it by writing spec PRs and stuff, but they also struggle to find these contributors.